src/App.tsx

```diff
@@ -1,24 +1,164 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import * as todosService from './utils/todos';
+import { NotificationText, SortCondition } from './types/enums';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { Header } from './components/Header/Header';
+import { USER_ID } from './utils/user';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[] | null>(null);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[] | null>(null);
+  const [filterBy, setFilterBy] = useState(SortCondition.All);
+  const [activeItemsCount, setActiveItemsCount] = useState(0);
+  const [completedItemsCount, setCompletedItemsCount] = useState(0);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [todosInProcess, setTodosInProcess] = useState<number[]>([]);
+
+  // #region Notifications
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)